app/workers/import_profile.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Workers
+  class ImportProfile < Base
+    sidekiq_options queue: :medium
+
+    include Diaspora::Logging
+
+    def perform(user_id)
+      user = User.find_by(username: user_id)
+      if user.nil?
+        logger.error "A user with name #{user_id} not a local user"
+      else
+        logger.info "Import for profile #{user_id} at path #{user.export.current_path} requested"
+        import_user_profile(user.export.current_path, user_id)
+
+        logger.info("Moving photos from impport file")
+        import_user_photos(user)
+      end
+    end
+
+private
```

Comment:
- Layout/AccessModifierIndentation: Indent access modifiers like `private`.
- Layout/EmptyLinesAroundAccessModifier: Keep a blank line before and after `private`.